orx-color/src/jvmDemo/kotlin/DemoSpectralZucconiColormap.kt

```diff
@@ -0,0 +1,24 @@
+import org.openrndr.application
+import org.openrndr.extra.color.colormaps.ColormapPhraseBook
+import org.openrndr.extra.color.colormaps.spectralZucconi6
+import org.openrndr.extra.noise.fastFloor
+import kotlin.math.sin
+
+fun main() = application {
+    program {
+        ColormapPhraseBook.register()
```

Comment:
`.register()` is only needed for `#pragma import`, right?

Reply:
You are right, it is not needed here at all. Fixed